py_mind_memo/view.py

```
@@ -686,3 +695,42 @@ def on_close():
+        # ファイルパスが設定されており、変更があり、かつ編集中・保存中でない場合のみ保存
+        if (self.persistence.current_file_path and 
+            self.model.is_modified and 
+            not self.editor.is_editing() and
+            not self._is_saving):
+            
+            self._is_saving = True
+            # メインスレッドでデータをキャプチャ
+            data = self.model.save()
+            file_path = self.persistence.current_file_path
+            
+            def run_save():
```

Ruff (0.15.7)

[warning] Missing return type annotation for private function `run_save`

Add return type annotation: `None`

(ANN202)